tests/unittests/GraphOptzTest.cpp

```diff
@@ -373,17 +373,21 @@ TEST_F(GraphOptz, optimizeBatchNormAfterConvWithPred) {
   EXPECT_EQ(F_->getNodes().size(), 3);
 
   ::glow::convertPlaceholdersToConstants(F_, bindings_, {});
-  ::glow::optimize(F_, CompilationMode::Infer);
-  EXPECT_EQ(F_->getNodes().size(), 2);
+  optimizedF_ = optimizeFunction(F_);
 
-  ASSERT_EQ(A->getNumUsers(), 1);
-  Node *newCV = A->getUsers().begin()->getUser();
+  EXPECT_EQ(optimizedF_->getNodes().size(), 2);
+
+  ASSERT_EQ(A->getNumUsers(), 2);
```

Comment:
This condition was comparing with 1 in the old version, now it is 2. Why?

Reply:
With new changes, when optimizedF_ = optimizeFunction(F_); is executed, number of users associated with Node A changes from 1 to 2.

static Function *optimizeFunction(Function *F) {
  auto *G = F->clone(F->getName().str() + "_optimized");
  ::glow::optimize(G, CompilationMode::Infer);
  return G;
}